src/components/paymentTable/paymentTable.scss

```diff
@@ -0,0 +1,14 @@
+.gray-text {
+  color: #3a3a3a;
```

Comment:
make the color as a variable 
every thing could be changed in the future should be at one place as a variable